github/orgs_codesecurity_configurations.go

```diff
@@ -143,12 +189,12 @@ func (s *OrganizationsService) DetachCodeSecurityConfigurationsFromRepositories(
 	return resp, nil
 }
 
-// GetCodeSecurityConfiguration gets a code security configuration available in an organization.
+// ListCodeSecurityConfiguration gets a code security configuration available in an organization.
 //
 // GitHub API docs: https://docs.github.com/rest/code-security/configurations#get-a-code-security-configuration
 //
 //meta:operation GET /orgs/{org}/code-security/configurations/{configuration_id}
-func (s *OrganizationsService) GetCodeSecurityConfiguration(ctx context.Context, org string, id int64) (*CodeSecurityConfiguration, *Response, error) {
+func (s *OrganizationsService) ListCodeSecurityConfiguration(ctx context.Context, org string, id int64) (*CodeSecurityConfiguration, *Response, error) {
```

Comment:
See the comment above:

```suggestion
func (s *OrganizationsService) GetCodeSecurityConfiguration(ctx context.Context, org string, id int64) (*CodeSecurityConfiguration, *Response, error) {
```